src/lib/network/clients/network_proposal.rb

```diff
@@ -120,23 +123,27 @@ def switch_backend_link
       # TRANSLATORS: information about the network backend in use. %s is the name of backend,
       # example "wicked" or "NetworkManager"
       backend_in_use = _("Using <b>%s</b>")
-      # TRANSLATORS: text of link to switch to another network backend. %s is the name of backend,
-      # example "wicked" or "NetworkManager"
-      switch_to = _("switch to %s")
 
-      if wicked_backend?
-        current_backend         = "wicked"
-        link_to_another_backend = Hyperlink(SWITCH_TO_NETWORK_MANAGER, switch_to % "NetworkManager")
+      case settings.current_backend
+      when :wicked
+        current_backend = "wicked"
+        links = [disable_link]
+        links.prepend(network_manager_link) if settings.network_manager_available?
+      when :network_manager
+        current_backend = "NetworkManager"
+        links = [wicked_link, disable_link]
       else
-        current_backend         = "NetworkManager"
-        link_to_another_backend = Hyperlink(SWITCH_TO_WICKED, switch_to % "wicked")
+        backend_in_use = "<b>%s</b>"
+        current_backend = _("Network Services Disabled")
+        links = [wicked_link]
+        links.append(network_manager_link) if settings.network_manager_available?
       end
 
-      "<ul><li>#{backend_in_use % current_backend} (#{link_to_another_backend})</li></ul>"
+      "<ul><li>#{backend_in_use % current_backend} (#{links.join(", ")})</li></ul>"
     end
 
     def launch_network_configuration(args)
-      log.info "Launching network configuration"
+      log.info "Launching network configuration NEW"
```

Comment:
NEW?

Reply:
It is a leftover used for debugging :stuck_out_tongue_closed_eyes: